webpack.config.development.js

```diff
@@ -15,7 +15,7 @@ const port = process.env.PORT || 3000;
 export default validate(merge(baseConfig, {
   debug: true,
 
-  devtool: 'cheap-module-eval-source-map',
+  devtool: 'inline-source-map',
```

Comment:
I'm getting much faster initial build and incremental build performance by using `cheap-inline-source-map`. May I ask why you chose `inline-source-map`?